### PR TITLE
Copyediting

### PR DESCRIPTION
**Pull request by bensteinberg**
This removes a few instances of the phrase "allows to".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ js-wacz create --file "collection/*.warc"
 
 ### --output, -o
 
-Allows to specify where the resulting `.wacz` file should be created, and what its filename should be.
+Specify where the resulting `.wacz` file should be created, and what its filename should be.
```
Comment (matteocargnelutti) [inline README.md:85]: My brain wants to use "Specifies" and I am not sure why 🤔 . What do you think @bensteinberg ?

**Review by matteocargnelutti — APPROVED** [review]

**Comment by matteocargnelutti** [comment]
Thank you @bensteinberg ! I left a question but this is much better

File: README.md (hunk 1 above)
Comment (bensteinberg) [reply README.md:85]: I could see it either way. There could be an implicit "This option": "(This option) Specifies" -- or an implicit "This option allows you to" or "With this option you can": "(This option allows you to) Specify".

Comment (bensteinberg) [reply README.md:85]: I think I incline toward the latter in the interest of saving precious characters ;) but I'm happy to change these.

Comment (matteocargnelutti) [reply README.md:85]: Nope, that makes sense to me, let's ship it :D ! thanks again

**Pull request merged**